.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2.1
+
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: cimg/node:16.15.1
+
+set_env: &set_env
+  name: Setup Environment Variables
+  command: |
+
+    echo 'Fetching Base Commit from GitHub'
+    echo 'export CIRCLE_PR_NUMBER="${CIRCLE_PR_NUMBER:-${CIRCLE_PULL_REQUEST##*/}}"' >> $BASH_ENV
+    source $BASH_ENV
+    echo "export CIRCLE_PR_BASE_SHA=`curl -s https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/pulls/${CIRCLE_PR_NUMBER} | jq -r '.base.sha'`" >> $BASH_ENV
+    echo 'export AFFECTED_ARGS="--base=${CIRCLE_PR_BASE_SHA}"' >> $BASH_ENV
+
+    source $BASH_ENV
+    echo $AFFECTED_ARGS
```

Comment:
Would you like to get the current PR base to commit SHA and save the `--base=${CIRCLE_PR_BASE_SHA}"` to `AFFECTED_ARGS`, then use it by `nx afftected` to lint changed part and test the changed part in next commit?

Maybe you could rename the `Setup Environment Variables` to hint at more messages or add a comment to explain why you write the `set_env`